spec/lib/sentry/processor/pii_sanitizer_processor_spec.rb

```diff
@@ -24,7 +24,11 @@
         gender: 'M',
         phone: '5035551234',
         va_eauth_birthdate: '1945-02-13T00:00:00+00:00',
-        va_eauth_pnid: '796375555'
+        va_eauth_pnid: '796375555',
+        accountType: 'SAVINGS',
```

Comment:
The Sentry docs say it works with nested data but could you update the spec to confirm that? The original data on staging is wrapped in a `"directDeposit"` object.